README.md

```diff
@@ -38,54 +29,16 @@ Do `docker run -p 80:80 -v $(pwd)/output:/usr/share/nginx/html nginx`
 ### Combined
 Do `rm -rf output && rm -rf temp/ && ./_genonce.sh && docker run -p 80:80 -v $(pwd)/output:/usr/share/nginx/html nginx`
 
-# \<Formal publications\>
```

Comment:
My impression was that this process is no longer followed, and it's fully automated. Am I correct in this assumption? If not, I'll revert 👍

Reply:
Correct.